src/avg.js

```diff
@@ -1,25 +1,20 @@
 let sum = require('./sum');
 
-/**
- * Returns the average of the numbers in the input array.
- *
- * Use your own sum function, required above. Assume that
- * the input array is non-empty.
- *
- * @param {number[]} array - A non-empty array of numbers
- * @returns {number} The average
- */
+
 function avg(array) {
-  // This is your job. :)
-  // Use the sum function you wrote rather than re-implementing
-  // it in this file. We required it above.
+  let sum = 0;
+  for(i = 0; i< array.length; i++){
```

Comment:
The idea here is to use the `sum` function you wrote in `sum.js`, which we required above.